pandas/tests/reductions/test_reductions.py

```diff
@@ -960,6 +960,18 @@ def test_min_max(self):
         assert np.isnan(_min)
         assert _max == 1
 
+        cat = Series(Categorical(
```

Comment:
these should go with the categorical tests not here

Reply:
Jeff, please looks at the rest of the test. This addition is very logical here (it is about categorical Series, not Categorical)

Reply:
at the very least make this a new test name